tests/core/GraphsTest.py

```diff
@@ -0,0 +1,147 @@
+from collections import Iterable
+from functools import partial
+import unittest
+
+from coalib.core.CircularDependencyError import CircularDependencyError
+from coalib.core.Graphs import traverse_graph
+
+
+def get_successive_nodes(graph, node):
+    try:
+        item = graph[node]
+    except KeyError:
+        return set()
+
+    if isinstance(item, Iterable):
+        return item
+    else:
+        return {item}
+
+
+def traverse_graph_on_cyclic_graph_test(graph, start_nodes):
```

Comment:
that's a function that generates tests, so I thought this convention is more appropriate^^